src/js/apps/forms/form/form-patient_app.js

```diff
@@ -36,6 +43,7 @@ export default App.extend({
     this.patient = patient;
     this.form = form;
     this.isReadOnly = this.form.isReadOnly();
+    this.currentUser = Radio.request('bootstrap', 'currentUser');
```

Comment:
might as well do this on line 30 in the `onBeforeStart` and use it in `initFormState`

Reply:
Ahhh, that's a good way to do it. I hemmed and hawed a lot on where to put that.